### ISSUE TITLE
Pie Chart: Value (%) label not being read by screenreaders

### ISSUE THREAD

**Issue by dreadnl** [visa/visa-chart-components#67]
Pie Chart: Value (%) label not being read by screenreaders

To replicate: 
-Turn on screenreader (voiceover/chrome/mac or NVDA/Chrome/Win11)
- Go to piechart example: https://visa.github.io/visa-chart-components/?path=/story/pie-chart--default#storybook-preview-wrapper
- Navigate to piechart slice
- Note that Value(%) value is not read out by screen readers

Thanks for sharing this great work!

**Comment by frankelavsky** [comment]
I can confirm this is happening on my end too. Seems like a bug, since the expected experience should include that percentage!

@dreadnl, There are some easy ways to resolve this in the meantime though, if you wanted! You can pre-calculate the percentages and pass them in as an additional property of each data point. like: { category: "United States", value: 147, percentage: "33%"} and then use the property accessibility = {…, elementDescriptionAccessor: "percentage" } to get the percentage to read out after the first two values. The property `elementDescriptionAccessor` of the `accessibility` prop is used to add any additional information to screen readers that is otherwise missing or unclear based on the input data alone.

**Comment by chris-demartini** [comment]
This is fixed in the most recent release, closing the issue.
